docs/user_manual/type_data.md

```diff
@@ -55,6 +55,16 @@ only be deleted if it is not used by any existing projects.
 
 
 
+## Deliverables
+
+A deliverable is the product that will be transferred to the client. This is set at the project level, and can be 
```

Comment:
"The" is correct because you can only choose one. If the project has multiple deliverables, you'd have to create a single "deliverable" option in MISO to represent the set. e.g. "Report and fastqs"

Reply:
Why not multiple deliverables in that situation?